[CI] SmokeTestMultiNodeClientYamlTestSuiteIT test {yaml=logsdb/10_settings/logsdb with default ignore dynamic beyond limit and subobjects false} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #4646 / amazonlinux-2_platform-support-aws](https://gradle-enterprise.elastic.co/s/aim5knoqiioya)
- [elasticsearch-periodic #4676 / openjdk21_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/yhtvd3niegab4)
- [elasticsearch-periodic-platform-support #4641 / rhel-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/dtd3f5ftlg4w6)
- [elasticsearch-periodic #4671 / openjdk21_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/sg6wsydhe2jzy)
- [elasticsearch-periodic #4671 / graalvm-ce17_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/3ypiaykapihtc)
- [elasticsearch-periodic-platform-support #4636 / sles-15_platform-support-unix](https://gradle-enterprise.elastic.co/s/ffnpzsw4ewdfq)
- [elasticsearch-periodic #4666 / openjdk17_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/uz6mqe3tr4cdk)

**Reproduction Line:**
```
./gradlew ":qa:smoke-test-multinode:yamlRestTest" --tests "org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT.test {yaml=logsdb/10_settings/logsdb with default ignore dynamic beyond limit and subobjects false}" -Dtests.seed=66CBC221C9024F71 -Dtests.locale=ks-Arab-IN -Dtests.timezone=America/Cayenne -Druntime.java=23
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Byaml%3Dlogsdb%2F10_settings%2Flogsdb%20with%20default%20ignore%20dynamic%20beyond%20limit%20and%20subobjects%20false%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [logsdb/10_settings:876]: 
Expected: "bar"
     but: was "foo"
```

**Issue Reasons:**
- [8.x] 2 consecutive failures in step openjdk21_checkpart1_java-matrix
- [8.x] 7 failures in test test {yaml=logsdb/10_settings/logsdb with default ignore dynamic beyond limit and subobjects false} (8.3% fail rate in 84 executions)
- [8.x] 2 failures in step openjdk21_checkpart1_java-matrix (66.7% fail rate in 3 executions)
- [8.x] 3 failures in pipeline elasticsearch-periodic-platform-support (100.0% fail rate in 3 executions)
- [8.x] 3 failures in pipeline elasticsearch-periodic (100.0% fail rate in 3 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -285,12 +285,6 @@ tests:
 - class: org.elasticsearch.indexing.IndexActionIT
   method: testAutoGenerateIdNoDuplicates
   issue: https://github.com/elastic/elasticsearch/issues/115716
-- class: org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT
-  method: test {yaml=logsdb/10_settings/logsdb with default ignore dynamic beyond limit and subobjects false}
-  issue: https://github.com/elastic/elasticsearch/issues/116054
-- class: org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT
-  method: test {yaml=logsdb/10_settings/logsdb with default ignore dynamic beyond limit and default sorting}
-  issue: https://github.com/elastic/elasticsearch/issues/116062
 - class: org.elasticsearch.xpack.application.connector.ConnectorIndexServiceTests
   issue: https://github.com/elastic/elasticsearch/issues/116087
 - class: org.elasticsearch.compute.operator.FilterOperatorTests

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/logsdb/10_settings.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/logsdb/10_settings.yml
@@ -704,6 +704,7 @@ logsdb with default ignore dynamic beyond limit and default sorting:
         body:
           query:
             match_all: {}
+          sort: "@timestamp"
 
   - match: { hits.total.value: 2 }
   - match: { hits.hits.0._source.name: "bar" }
@@ -765,15 +766,16 @@ logsdb with default ignore dynamic beyond limit and non-default sorting:
         body:
           query:
             match_all: {}
+          sort: "@timestamp"
 
   - match: { hits.total.value: 2 }
-  - match: { hits.hits.0._source.name: "foo" }
-  - match: { hits.hits.0._source.value: 10 }
-  - match: { hits.hits.0._source.message: "the quick brown fox" }
+  - match: { hits.hits.0._source.name: "bar" }
+  - match: { hits.hits.0._source.value: 20 }
+  - match: { hits.hits.0._source.message: "jumps over the lazy dog" }
   - match: { hits.hits.0._ignored: [ "host", "message", "pid", "region", "value" ] }
-  - match: { hits.hits.1._source.name: "bar" }
-  - match: { hits.hits.1._source.value: 20 }
-  - match: { hits.hits.1._source.message: "jumps over the lazy dog" }
+  - match: { hits.hits.1._source.name: "foo" }
+  - match: { hits.hits.1._source.value: 10 }
+  - match: { hits.hits.1._source.message: "the quick brown fox" }
   - match: { hits.hits.1._ignored: [ "host", "message", "pid", "region", "value" ] }
 
 ---
@@ -870,6 +872,7 @@ logsdb with default ignore dynamic beyond limit and subobjects false:
         body:
           query:
             match_all: {}
+          sort: "@timestamp"
 
   - match: { hits.total.value: 2 }
   - match: { hits.hits.0._source.name: "bar" }

